### ISSUE TITLE
Strange Behaviour in Linux terminal of the interactive

### ISSUE THREAD

**Issue by julien-leclercq** [fsprojects/Forge#114]
## Expected Behavior
1. Run the interactive 
2. Make the modification to my project with comfort

## Current Behavior
1. Run the interactive.
2. Displays strange characters and double the displaying of every typing + displays the backarrow character in command line instead of erasing what I typed

## Possible Solution
I think the problems come from uncatched system signals or something like that

## Steps to Reproduce (for bugs)
1. run build.sh (on linux)
2. run few commands to scaffold your project
3. stare with admiration at all the random character displayed

**Comment by banshee** [comment]
I'm not seeing that behavior, but It's ignoring control-U for me.  That's actually a similar symptom; I think there's something weird about the terminal mode it's using for interactive mode.

**Comment by kerams** [comment]
Could it be caused by https://github.com/fsprojects/Forge/blob/master/src/Forge/Program.fs#L59? I'm getting ╬╗ instead of the lambda on Windows :).

**Comment by julien-leclercq** [comment]
@kerams no, the lambda displays right, I realized last week that it happens when a command fail, so I think there is a bad behaviour of the interaction loop when the return code is not 0

**Comment by amcguier** [comment]
I can consistently duplicate this behavior in iterm2 in osx even when the command is successful
**iterm version:** 3.0.10
**Forge version:** 1.0.0  - from homebrew

`forge`
 `update paket`

The behavior occurs on other commands too (such as project creation) but not as consistently.
